replicators/src/noria_adapter.rs

```diff
@@ -244,6 +246,13 @@ impl NoriaAdapter {
                     tokio::time::sleep(WAIT_BEFORE_RESNAPSHOT).await;
                     resnapshot = true;
```

Comment:
I think we probably want to also set `full_snapshot` to `false` here

Reply:
Done